woocommerce/class-sv-wc-plugin.php

```diff
@@ -790,6 +814,19 @@ public function is_tls_1_2_available() {
 	}
 
 
+	/**
+	 * Determines whether the plugin supports HPOS.
+	 *
+	 * @since x.y.z
+	 *
+	 * @return bool
+	 */
+	public function is_hpos_compatible() : bool
+	{
+		return $this->supports_hpos;
```

Comment:
TODO: we might need to add a WC version check here for when this bug has been addressed: https://github.com/woocommerce/woocommerce/issues/35703